### PR TITLE
[需要志愿者review] `docs/developer-guide/issues.md` 中文翻译

### PR DESCRIPTION
**Pull request by gucong3000**
> Which issue, if any, is this issue related to?

e.g. "Closes #000" or "None, as it's a documentation fix."

> Is there anything in the PR that needs further explanation?

e.g. "No, it's self explanatory."

### CODE REVIEW

File: docs/developer-guide/issues.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+        -   "Add y" 例如："Add unit-blacklist"
+        -   "Add x to y" 例如："Add ignoreProperties: [] to property-blacklist"
+        -   "Fix false positives/negatives for x in y" 例如："Fix false positives for Less mixins in color-no-hex"
+    -   使用 `*` 如果这个问题适用于一组规则，例如："Fix false negatives for SCSS variables in selector-*-pattern"
+-   提供一个指向[开发者指南](../developer-guide.md)中相关部分的链接，当：
+    -   添加 `help wanted` 标签来鼓励楼主去贡献提交 例如：[向现有规则添加选项](../developer-guide/rules.md#adding-an-option-to-an-existing-rule)或者[修复现有规则中的 bug](../developer-guide/rules.md#fixing-a-bug-in-an-existing-rule)
+    -   关闭一个问题，因为该功能最好是生态系统的一部分 例如：[插件](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md) 或者 [处理器](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/processors.md)
+-   仅在问题上使用里程碑，而不要在拉取请求上使用里程碑，并且：
+    -   为引入重大变化的问题使用 `future-major` 里程碑
+    -   可选项，创建版本里程碑（例如`8.x`）来管理即将发布的版本
```
Comment (Xiaohanxu1996) [inline docs/developer-guide/issues.md]: 数字8之前添加空格（中文与数字之间需要添加空格）

File: docs/developer-guide/issues.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
 ```
 
-There are three rules of thumb. You should use the:
+这里有三个何时使用何种标签的经验法则：
 
--   `status: discussion`, `status: needs clarification` or `status: needs investigation` label when first triaging an issue
--   `help wanted`, a `type` (and `non-standard syntax: *` and `good first issue`) labels when a course of action is agreed
--   `status: wip` label when your are, or someone has said they are, about to start working on an issue
+-   首次对问题进行分类时：`status: discussion`、`status: needs clarification` 或者 `status: needs investigation`
+-   当行动方案达成一致时： `help wanted`, `type` (还有 `non-standard syntax: *` 和 `good first issue`)
```
Comment (Xiaohanxu1996) [inline docs/developer-guide/issues.md]: 此行英文字母help之前的空格数与上一行字母前 及本行其余字母前空格数不一致。

File: docs/developer-guide/issues.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
--   provide a link to the relevant section of the [Developer Guide](../developer-guide.md) when:
-    -   adding the `help wanted` label to encourage the original poster to contribute, e.g. [adding an option to an existing rule](../developer-guide/rules.md#adding-an-option-to-an-existing-rule) or [fixing a bug in an existing rule](../developer-guide/rules.md#fixing-a-bug-in-an-existing-rule)
-    -   closing an issue because the feature is best part of ecosystem e.g. [a plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md) or [processor](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/processors.md)
--   use milestones only on issues and not on pull requests and:
-    -   use the `future-major` milestone for issues that introduce breaking changes
-    -   optionally, create version milestones (e.g. `8.x`) to manage upcoming releases
--   use the following saved reply to close any issue that do not use the template:
+-   使用[标签](https://github.com/stylelint/stylelint/labels)，并且：
+    -   添加 _一个_ `status: *` 标签 (或者在准备好的时候添加 `help wanted` 标签)
```
[the remark this answers is not in the document]
Comment (Xiaohanxu1996) [reply docs/developer-guide/issues.md]: line 6 最后面的闭括号

File: docs/developer-guide/issues.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
-# Managing issues
+# 管理问题
 
-You should:
+您可以:
```
[the remark this answers is not in the document]
Comment (gucong3000) [reply docs/developer-guide/issues.md]: 已改正

File: docs/developer-guide/issues.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (gucong3000) [reply docs/developer-guide/issues.md]: 已改正

File: docs/developer-guide/issues.md (hunk 1 above)
Comment (gucong3000) [reply docs/developer-guide/issues.md]: 已添加一个空格

File: docs/developer-guide/issues.md (hunk 2 above)
Comment (gucong3000) [reply docs/developer-guide/issues.md]: 已修改为
```md
-   当行动方案达成一致时：`help wanted`、`type` (还有 `non-standard syntax: *` 和 `good first issue`)
```

**Pull request merged**